docs/source/examples.rst

```diff
@@ -0,0 +1,43 @@
+Examples
```

Comment:
Hi Tristan,
It isn't clear what the examples below are used for. Are these parameters to a method or a data structure returned? Please provide some context.
Thanks!

Reply:
Added more description to make it clear they are examples of arguments given to the ModelCreator.

Reply:
We chatted through this in person and got some of that sweet alignment. :)